thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/AutotuneConfigBean.java

```diff
@@ -143,12 +157,13 @@ public boolean equals(Object o){
         && Objects.equals(getConfiguration(), fac.getConfiguration()) && Objects.equals(getPerformance(), fac.getPerformance())
         && Objects.equals(getPerformanceEvaluationMethod(), fac.getPerformanceEvaluationMethod())
         && Objects.equals(getAvgRunningTime(), fac.getAvgRunningTime())
-        && Objects.equals(getOverallRunningTime(), fac.getOverallRunningTime());
```

Comment:
If we are going to reposition this class to host autotune (tune alert filter or later alert-group filter), let's redesign this class carefully.